aries_cloudagent/anoncreds/anoncreds/routes.py

```diff
@@ -0,0 +1,434 @@
+"""Anoncreds admin routes."""
+import logging
+
+from aiohttp import web
+from aiohttp_apispec import (
+    docs,
+    match_info_schema,
+    querystring_schema,
+    request_schema,
+    response_schema,
+)
+from marshmallow import fields
+
+# from ...admin.request_context import AdminRequestContext
+from ...messaging.models.openapi import OpenAPISchema
+from ...messaging.valid import (
+    GENERIC_DID,
+    INDY_CRED_DEF_ID,
+    INDY_SCHEMA_ID,
+    INDY_VERSION,
+    UUIDFour,
+)
+
+LOGGER = logging.getLogger(__name__)
+
+SPEC_URI = ""
+
+schemaId = fields.Str(
+    data_key="schemaId", description="Schema identifier", **INDY_SCHEMA_ID
+)
+
+
+class SchemaIdMatchInfo(OpenAPISchema):
+    """Path parameters and validators for request taking schema id."""
+
+    schema_id = schemaId
+
+
+class CredIdMatchInfo(OpenAPISchema):
+    """Path parameters and validators for request taking credential id."""
+
+    cred_def_id = fields.Str(
+        description="Credential identifier", required=True, example=UUIDFour.EXAMPLE
+    )
+
+
+schemaAttrNames = fields.List(
+    fields.Str(
+        description="Attribute name",
+        example="score",
+    ),
+    description="Schema attribute names",
+    data_key="attrNames",
+)
+schemaName = fields.Str(
+    description="Schema name",
+    example=INDY_SCHEMA_ID["example"].split(":")[2],
+)
+schemaVersion = fields.Str(description="Schema version", **INDY_VERSION)
+issuerId = fields.Str(
+    description="Issuer Identifier of the credential definition or schema",
+    **GENERIC_DID,
+)  # TODO: get correct validator
+
+
+class SchemaSchema(OpenAPISchema):
+    """Marshmallow schema for indy schema."""
+
+    attrNames = schemaAttrNames
+    name = schemaName
+    version = schemaVersion
+    issuerId = issuerId
```

Comment:
https://github.com/hyperledger/aries-cloudagent-python/blob/main/aries_cloudagent/messaging/decorators/attach_decorator.py#L41-L72

For the models that were defined in #108, we should have model-schema pairs like those in the example above.